ios/Podfile

```diff
@@ -0,0 +1,30 @@
+source 'https://github.com/CocoaPods/Specs.git'
+platform :ios, '9.0'
+
+target 'MagnetJourney' do
+  # Uncomment the next line if you're using Swift or would like to use dynamic frameworks
+  # use_frameworks!
+  
+  pod 'React', path: '../node_modules/react-native', :subspecs => [
```

Comment:
The React libraries are no longer added to the 'Build Phases' tab and defined here instead. I copied the airbnb-maps example app.